ocw/lib/db.py

```diff
@@ -67,6 +67,15 @@ def sync_csp_to_local_db(pc_instances, provider, namespace):
         update(state=StateChoice.DELETED)
 
 
+def tag_to_boolean(tag_name, csp_info):
+    if 'tags' not in csp_info:
+        return False
+    if tag_name in csp_info['tags']:
+        return bool(csp_info['tags'][tag_name])
+    else:
+        return False
```

Comment:
```suggestion
    try:
        return bool(csp_info['tags'][tag_name])
    except KeyError:
        return False
```

Reply:
using exceptions as flow :( its expensive

Reply:
but for line 73-76 can be used `get`:
```
return bool(cap_info['tags'].get(tag_name, False)
```